examples/durable/DurableApp/MonitorStart/run.ps1

```diff
@@ -0,0 +1,13 @@
+param($Request, $TriggerMetadata)
+
+Write-Host 'MonitorStart started'
+
+$OrchestratorInputs = @{ JobId = $null; PollingInterval = 10; ExpiryTime = (Get-Date).addSeconds(60).ToUniversalTime() }
```

Comment:
You need to convert to UTC first, and then add the 60 seconds. The order matters: local time is ambiguous and not continuous, so the results of arithmetic operations can be surprising (for example, on the nights of switching to/from DST).